### PR TITLE
fix: Home app list show style

### PR DESCRIPTION
**Pull request by whDongRui**

### CODE REVIEW

File: src/components/AppList/index.scss [hunk 1]
```diff
@@ -1,25 +1,26 @@
 @import '~scss/vars';
 
-.appList {
-  a {
+
+.appList, .categoryApps {
+  overflow: hidden;
+  _zoom: 1;
```
Comment (iwisunny) [inline src/components/AppList/index.scss]: `_zoom` ? typo

File: src/components/AppList/index.scss [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
   }
 }
-.categoryApps{
-  margin-top: -16px;
-}
 
-.appsRow {
-  display: flex;
-  justify-content: space-between;
+.categoryApps {
```
Comment (iwisunny) [inline src/components/AppList/index.scss]: simple way is using flex, and remove `justify-content`

File: src/components/AppList/index.scss (hunk 1 above)
Comment (whDongRui) [reply src/components/AppList/index.scss]: Clear float style ,  _zoom for compatible IE,  detail see https://www.cnblogs.com/java-zone/articles/2733991.html.

File: src/components/AppList/index.scss (hunk 2 above)
Comment (whDongRui) [reply src/components/AppList/index.scss]: If using flex, 6 or more apps will show one line.

Comment (iwisunny) [reply src/components/AppList/index.scss]: OK

**Pull request merged**